sovrin_client/cli/cli.py

```diff
@@ -1305,32 +1305,43 @@ def _connectTo(self, matchedVars):
                 self.print(envError, token=Token.Error)
                 self._printConnectUsage()
```

Comment:
This section will prevent the command parsing logic to stop here since it does not return True. You can simply move the `return True` outside of the check for valid env name and node reg.